tests/unit/FileUploadPreview.spec.ts

```diff
@@ -211,20 +200,19 @@ describe('FileUploadPreview component', () => {
       propsData: { pdfPageSize: PdfPageSize.LETTER_SIZE },
       vuetify
     })
+
     const fileInput = wrapper.find('input[type="file"]')
     setupFileInput(fileInput)
     inputValue = invalidSecondPagePdf.name
     inputFilesGet.mockReturnValue([invalidSecondPagePdf])
-    fileInput.trigger('change')
-
+    await fileInput.trigger('change')
     await waitForUpdate(5)
```

Comment:
I tried with a count of 3 but it worked intermittently in the GH CI check, so 5 it is.